client/src/components/NationwideMap.vue

```diff
@@ -267,18 +271,26 @@ export default defineComponent({
     setUpControls(): void {
       if (this.map == null) return;
 
+      const geolocateControl = new mapboxgl.GeolocateControl({
+        positionOptions: {
+          // If enabled, gets the best possible results. Can result in slower response times or
+          // increased power consumption so set to false for now.
+          enableHighAccuracy: false,
+        },
+        // Receive updates to the device's location as it changes.
+        trackUserLocation: true,
+      });
+
+      geolocateControl.on('geolocate', (result: any) => {
+        const lat = result?.coords?.latitude;
+        const long = result?.coords?.longitude;
+        const baseUrl = this.scorecard ? SCORECARD_BASE : MAP_ROUTE_BASE;
+
+        this.$router.push(`${baseUrl}/${GeoType.address}/${lat},${long}`);
```

Comment:
Can `lat` or `long` be undefined here? (I don't really understand the `?` notation in ts yet) If so, let's make sure they have a value before pushing `undefined` to the URL.

Reply:
Done!